client/src/components/DescriptionView.jsx

```diff
@@ -1,11 +1,22 @@
 import React from 'react';
+import styled from 'styled-components';
+import PriceView from './PriceView.jsx';
+import GrayLine from './GrayLine.jsx';
 
+// const primeURL = 
+
+const Description = styled.div`
+  font-size: 13px;
+`;
 const Paragraph = ({ paragraph }) => (<p>{ paragraph }</p>);
 
-const DescriptionView = ({ description }) => (
-  <div style={{fontSize: 13}}>
+const DescriptionView = ({ price, prime, description}) => (
```

Comment:
I noted that in most of your components, you deconstruct the incoming props object into variables. I quite like this approach, it feels much less convoluted than referring to the properties of a props object.

Reply:
Yeah it looks nicer too!